src/goto-harness/memory_snapshot_harness_generator.cpp

```diff
@@ -111,7 +111,7 @@ void memory_snapshot_harness_generatort::validate_options(
   {
     throw invalid_command_line_argument_exceptiont(
       "option --memory_snapshot is required",
-      "--harness-type initialise-from-memory-snapshot");
+      "--harness-type initialise-with-memory-snapshot");
```

Comment:
`initialise-with-memory-snapshot` -> `initialize-with-memory-snapshot` (to be consistent with AE spelling we use everywhere else) 🇺🇸

Reply:
I've added a separate commit fixing this across code, documentation, tests.